### DISCUSSION TITLE
Battery Voltage limitation

### DISCUSSION THREAD

**Discussion by ericgrand** [syssi/esphome-soyosource-gtn-virtual-meter#162] [Q&A]
Hello!
I have 6 valence batteries U27-12XP, 138Ah 12V LFP each. I'm currently using them in 24V mode feeding 2 enphase M215 micro inverter to limit the grid consumption when price is high.
I'm considering switching to a more versatile setup with a drivable inverter to limit power and not export to grid from battery.
I want the solution to be low cost (batteries have been salvaged from a dead car).
My question is on the various versions of the Soyosource inverters.
I can either go for:
-a 1000W for 24V battery (high amps and limited charging capability)
-a 1000W for 36V battery, less Amp but still limited charging capability
-a 12000W for 72V version, which is ideal I believe on the paper.
My question is related to the 2 modes of the Soysource inverters. Can I use batteries in PV mode? Will the inverter draw as much amp as possible or will it limit the amps to be at the max power set?
I believe it should be more efficient to have a 55-90V inverters fed with batteries between 72 and 84V. But this model is given for 48V battery...
Can someone give me some inputs on how the inverter would behave with batteries connected instead of PV?

Thanks in advance!

**Comment by syssi** [comment]
Did you see this discussion? https://github.com/syssi/esphome-soyosource-gtn-virtual-meter/discussions/145

**Comment by ericgrand** [comment]
Yes, but unfortunately, it is not precised which model was used. Also, the use of a DC-DC converter in the middle allow to limit the amount of amps the inverter can draw. Connected batteries directly may be an issue if the inverter has no current limiter on input.
DC-DC converter will act like a solar panel, while MPPT will try to draw more amps, the voltage will sag. With battery capable of 1000A instantaneous, it could be tricky...
